symengine/__init__.py

```diff
@@ -55,7 +55,9 @@
 
     def lambdify(args, exprs, **kwargs):
         return Lambdify(args, *exprs, **kwargs)
-
+else:
+    def lambdify(args, exprs, **kwargs):
+        raise ModuleNotFoundError("Cannot import numpy, which is required for `lambdify` to work")
```

Comment:
Use a `__getattr__` method here and raise an `AttributeError` with this error message?

Reply:
Oh, didn't know you could. Today I learned.. since 3.7 apparently.